publish-test.sh

```diff
@@ -33,6 +33,14 @@ if [ "$?" != "0" ]; then
 	exit 1
 fi
 
+./gradlew --no-daemon -PlocalNexus -P${1} :extensions:remote-deploy-command:publish
```

Comment:
```suggestion
./gradlew --no-daemon -PlocalNexus -P${1} :extensions:remote-deploy-command:publish --info
```
So we can see the publishing happening in the log.

Reply:
Lets move these lines under the maven profile checking code that is below it, just to separate it out.